python_src/src/lamp_py/postgres/postgres_utils.py

```diff
@@ -404,52 +403,16 @@ def vacuum_analyze(self, table: Any) -> None:
             cursor.execute(sa.text("END TRANSACTION;"))
             cursor.execute(sa.text(f"VACUUM (ANALYZE) {table_as};"))
 
-    def add_metadata_paths(self, paths: List[str]) -> None:
-        """
-        add metadata filepaths to metadata table for testing
-        """
-        print(paths)
-        with self.session.begin() as session:
-            session.execute(
-                sa.insert(MetadataLog.__table__),
-                [{"path": p} for p in paths],
-            )
```

Comment:
moved this out of db manager class since it only makes sense for one of the databases.